ImageLoader/JPEGImageOperations/JPEGImageOperations.swift

```diff
@@ -0,0 +1,151 @@
+import Foundation
+import TurboJPEG
+ 
+#if os(iOS) || os(macOS) || os(tvOS) || os(watchOS)
+    import Darwin
+#elseif os(Android) || os(Linux)
+    import Glibc
+#elseif os(Windows)
+    import ucrt
+#else
+    #error("C library not known for the target OS")
+#endif
+
+public enum pixelFormat: Int32 {
```

Comment:
```suggestion
public enum PixelFormat: Int {
```

In general, with Swift, you should use `Int` unless you explicitly need it to be 32-bit in width for some reason.  Enumerated types should use PascalCase.